stm32mp1.mk

```diff
@@ -119,10 +119,11 @@ tfa-clean:
 # U-Boot
 ################################################################################
 U_BOOT_EXPORTS ?= CROSS_COMPILE="$(CCACHE)$(AARCH32_CROSS_COMPILE)"
+STM32MP1_DTS_U_BOOT ?= $(STM32MP1_DTS_BASENAME)
```

Comment:
Sure, but I'm confused as to which line 51 you mean, since that variable doesn't match any of the existing headers.
Do you mean after `U_BOOT_DTB` or after `TFA_PATH`?

Reply:
sorry, I meant 31:
```patch
 ifeq ($(PLATFORM),stm32mp1-157A_DK1)
 BREXT_FLAVOR		= STM32MP157A-DK1
 STM32MP1_DTS_BASENAME	= stm32mp157a-dk1
 else ifeq ($(PLATFORM),stm32mp1-157C_DK2)
 (...)
 else
 $(error Unknown PLATFORM $(PLATFORM))
 endif
 
+STM32MP1_DTS_U_BOOT ?= $(STM32MP1_DTS_BASENAME)
```